src/main/java/liquibase/addUserLevelFhirFlag.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+
+    <changeSet author="mat_dev_user" id="1" context="prod">
+        <sql>
```

Comment:
If you do it like this it will run even if the column is there so we don't have to comment it out manually in local.    
```
<changeSet author="mat_dev_user" id="3" context="prod">
        <preConditions onFail="MARK_RAN">
            <not><columnExists tableName="MEASURE_EXPORT" columnName="FHIR_LIBS_JSON"/></not>
        </preConditions>
        <addColumn tableName="MEASURE_EXPORT">
            <column name="FHIR_LIBS_JSON" type="longtext"/>
        </addColumn>
    </changeSet>
```

Reply:
Cool I am working on making these changes